frontend/src/general/LogoutMenu/__tests__/LogoutMenu.test.jsx

```diff
@@ -0,0 +1,197 @@
+import React from "react";
+import Enzyme from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import { beforeEach, describe, expect, it, jest } from "@jest/globals";
+
+import {
+  ClickAwayListener,
+  Grow,
+  MenuItem,
+  MenuList,
+  Paper,
+  Popper,
+  Typography
+} from "@material-ui/core";
+import { NavLink } from "react-router-dom";
+import LogoutMenu from "../LogoutMenu";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("<LogoutMenu/> functional component", () => {
+  let wrapper;
+  const dummyAnchor = { thing: "thing" };
+  const dummyOpen = true;
+  let mockHandleClose;
+  let mockHandleLogout;
+
+  describe("returns a component that", () => {
+    beforeEach(() => {
+      mockHandleClose = jest.fn();
+      mockHandleLogout = jest.fn();
+      wrapper = Enzyme.shallow(
+        <LogoutMenu
+          anchor={dummyAnchor}
+          open={dummyOpen}
+          handleClose={mockHandleClose}
+          handleLogout={mockHandleLogout}
+        />
+      );
+    });
```

Comment:
There's no afterEach() block to unmount the wrapper and to clear all jest mocks

Reply:
603d446fb5013d9040ea82d1b965d5f644397026